ekingan/day3.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    	<title>Day 3 Homework</title>
+        <link rel='stylesheet' type='text/css' href='stylesheet.css'/>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+        <script type='text/javascript' src='script.js'></script>
+	</head>
+	<body>
+    <h1> It is 10pm and I am still working on my homework. What should I do?</h1>
```

Comment:
Take a break.  Banging your head against the same problem doesn't always solve it.  Sometimes going for a run or taking a walk or a nap can really truthfully help.  I don't know how many times I've been like "almost got it" for 2 hours straight and finally take a break; then when I come back I realize I really was overthinking the problem or suddenly a new approach pops into my mind.